### PR TITLE
Checking linters

### PR DESCRIPTION
**Pull request by JbirdL86**
Creating pull request for final touches with linters.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,983 @@
...
+                                            </figure>
+                                        </a>
+                                    </div>
+                                    <div class="thumbnail-text">
+                                        <a href="https://static01.nyt.com/images/2020/07/16/science/16SOLARORBITER-loop-cover/16SOLARORBITER-loop-cover-threeByTwoSmallAt2X.jpg?quality=75&auto=webp&disable=upscale">
+                                            <h3 class="thumbnail-header">Step Inside a Spiraling Watch Museum in the Swiss Jura Mountains</h3>
+                                        </a>
+                                        <div>
+                                            <img src="https://tpc.googlesyndication.com/simgad/7430774282595300108?" alt="ads" style="vertical-align:bottom">
+                                        </div>
```
Comment (HADDADSOHAIB) [inline index.html:480]: Inline styles, remove this.

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
### Status: Change Required :recycle: 

### Hi @JbirdL86  :hand: 

### You did a fantastic job with the project, keep the good work.

### There are some changes that are required:
- You did not implement the bottom sticky footer:
Screenshot from 2020-07-18 18-42-23
- the readme file should have .md extension not .text, please fix that so the content will be displayed prporely.
Screenshot from 2020-07-18 18-45-52
As you can see the readme file is not displayed because of that.
- please remove inline styles, check my comments.
### once done with the changes, ask for another code review.

### Status: Change Required :recycle:

**Review by kubilaycaglayan — CHANGES_REQUESTED** [review]
Hello Team 🌞 ,
Status: Changes Required 🟠 

Great job on doing the changes. ✊
After doing a few small arrangements, you are good to go. 🚄 

- [ ] **Broken live demo link**
When we are creating live demo links on raw.githack.com we should use the link of the HTML file itself, not the repository or branch. Here, you will see where we are being redirected to when clicking the link.
wrong link
demo link

- [ ]  **Typo in the bottom sticky footer**
concatenated string

**[OPTIONAL]**   **Unprofessional usage message**
I believe we can try our best to make our readme files looks more professional. 
Usage parts of our readme file are more about the content and how to interact with it. In this project, we are creating a very valuable news source by positioning the elements. 😊 This is very helpful for other people.
usage message

That's it, thanks for creating this amazing project! 👍 

You can reach out to me via slack with my name.

With all my respect...

**Review by kubilaycaglayan — APPROVED** [review]
Hello Team 🌞 

**Great job on doing the changes.**

**Your project is approved 🟢** 

With all my respect...

**Comment by JbirdL86** [comment]
Thank you very much for your reviews and your quick response, you are
great! Motivational messages were great also!

**Pull request merged**